recipes/macdylibbundler/all/test_package/conanfile.py

```diff
@@ -0,0 +1,13 @@
+from conan import ConanFile
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "build_type"
+    generators = "VirtualBuildEnv"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.tool_requires(self.tested_reference_str)
+
+    def test(self):
+        self.run("dylibbundler -h")
```

Comment:
You might have some issues with corssbuilding scenarios with this.
probably a simpler check/guarding it could fix it :)

Reply:
Guess you mean `can_run` maybe? Just added it